src/compat/posix/include/sys/sysctl.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file
+ * @brief
+ *
+ * @date 2.12.2020
+ * @author Suraaj K S
+ */
+
+#ifndef COMPAT_POSIX_SYS_SYSCTL_H_
+#define COMPAT_POSIX_SYS_SYSCTL_H_
+
+#include <sys/cdefs.h>
+
+__BEGIN_DECLS
+
+struct sysct {
+	const char *processor;
+	const char *platform; 
+	const char *system;
+};
+
+
+extern int sysctl(struct sysct *);
```

Comment:
This looks good, but are you sure it's POSIX? ( I cannot find this here - https://pubs.opengroup.org/onlinepubs/9699919799/). If it's not POSIX may be move this code to `src/lib` for example.

Reply:
Okay, I will do so

Reply:
Thanks. Can I also ask you to push two commits instead of this one? The first one is about adding `sysctl` library, and another one is about using this library for `uname` command. This way makes the commits more atomic, independent, and easy to review.

Reply:
So, do I add the corresponding .h file in src/lib as well? or add it in one of the include directories?

Reply:
Yes, you can add .h file in the same directory along with sysctl.c and then use IncludeExport to export this .h file to be accessed further from uname command.

```
$ grep -nri IncludeExport src/lib/
```